src/store/cockpitApi.ts

```diff
@@ -89,8 +91,34 @@ export const cockpitApi = emptyCockpitApi.injectEndpoints({
           }
         },
       }),
+      deleteBlueprint: builder.mutation<
+        DeleteBlueprintApiResponse,
+        DeleteBlueprintApiArg
+      >({
+        // invalidatesTags: [{ type: 'Blueprints' }],
+        queryFn: async ({ id: filename }) => {
+          try {
+            const path = await getBlueprintsPath();
+            const filepath = path + '/' + filename;
```

Comment:
Wouldn't it be safer to use `path.join()`?

Reply:
Ya whoops, it would be a lot better